etc/templates/verify/exasol.tmpl

```diff
@@ -1,3 +1,5 @@
 -- Verify [% project %]:[% change %] on [% engine %]
 
 -- XXX Add verifications here.
+
+ROLLBACK;
```

Comment:
I think It'd keep the ROLLBACK here; All the other engines do that (plus have a BEGIN at the top). Good practice for verify scripts.